demos/dialog/index.js

```diff
@@ -34,3 +34,17 @@ window.toggleActions.onclick = function() {
   dialog.hideActions = hideAction;
   hideActionSpan.innerText = hideAction;
 };
+
+const textField = document.getElementById('dialog8-text-field');
```

Comment:
Use `querySelector('#id')` instead of `getElementById('id')`

Reply:
kewl kewl

Reply:
done